### PR TITLE
Frnt/comment

### PR DESCRIPTION
**Pull request by Dan767**
Adding comment field to display below post

### CODE REVIEW

File: distributedsocialnetwork/post/views.py [hunk 1]
```diff
@@ -33,7 +34,20 @@ def create_post(request):
 
 def view_post(request, pk):
     context = {}
+    if request.POST:
+        form = CommentCreationForm(request.POST)
```
Comment (ColonelSanders21) [inline distributedsocialnetwork/post/views.py]: I checked out this branch locally, it seems like CommentCreationForm does not exist. Was this an older name for PostCommentForm?

File: distributedsocialnetwork/post/views.py [hunk 2]
```diff
@@ -33,7 +34,20 @@ def create_post(request):
...
+        if form.is_valid():
+            new_comment = form.save(commit=False)
+            new_comment.author_id = user.id
```
Comment (ColonelSanders21) [inline distributedsocialnetwork/post/views.py]: Since this is in a different view, user is undefined. We just have to set user=request.user first.

**Review by ColonelSanders21 — CHANGES_REQUESTED** [review]
Couple of errors popping up when I check this out locally regarding CommentCreationForm and not setting user=request.user in view_post().

File: distributedsocialnetwork/post/views.py (hunk 1 above)
Comment (Dan767) [reply distributedsocialnetwork/post/views.py]: Whoops, that is my bad.  Thanks for catching that. This is corrected now in the latest commit.

File: distributedsocialnetwork/post/views.py (hunk 2 above)
Comment (Dan767) [reply distributedsocialnetwork/post/views.py]: This is corrected as well now, thanks.

**Comment by ColonelSanders21** [comment]
I made a couple more fixes. I feel more comfortable merging this, if someone else can take a quick glance as a sanity check that would be great.

**Review by JoePotentier — APPROVED** [review]

**Pull request merged**